etcdserver/api/v3rpc/member.go

```diff
@@ -95,6 +95,18 @@ func (cs *ClusterServer) MemberUpdate(ctx context.Context, r *pb.MemberUpdateReq
 	return &pb.MemberUpdateResponse{Header: cs.header()}, nil
 }
 
+func (cs *ClusterServer) MemberLeader(ctx context.Context, r *pb.MemberLeaderRequest) (*pb.MemberLeaderResponse, error) {
+	membs := cs.cluster.Members()
+	var peerURLs []string
+	if len(membs) > 0 {
+		peerURLs = membs[0].PeerURLs
```

Comment:
It's sorted by ID, so I think it's not always (https://github.com/coreos/etcd/blob/master/etcdserver/member.go#L160-L165).